components/atom/card/src/index.js

```diff
@@ -35,7 +35,7 @@ const AtomCard = ({
   tabIndex
 }) => {
   const redirectToHref = () => {
-    if (href) window.location.href = href
+    if (href) window.open(href, '_blank')
```

Comment:
this change suposes a major version. Could you please leave the default component behavior as is and only affect to the triggered demo with some flag?